lib/Horde/Core/Db/Migration.php

```diff
@@ -65,20 +65,59 @@ public function __construct($basedir = null, $pearconf = null)
         error_reporting($old_error_reporting & ~E_DEPRECATED);
         $pear = new PEAR_Config($pearconf);
 
+        // Detect and handle the Composer use case
+        if (class_exists('Composer\Autoload\ClassLoader')) {
```

Comment:
Add second parameter to prevent accidental loading - it should already be loaded for this scenario